d2l-button-icon.html

```diff
@@ -24,7 +24,7 @@
 
 				--button-rtl: {
 					left: 0;
-					right: calc(((var(--d2l-button-icon-min-width) - 0.9rem) / 2) * -1);
+					right: calc(((2rem + 2px - 0.9rem) / 2) * -1);
```

Comment:
I think it would be useful to allow this to be overridden. Perhaps a `--d2l-button-icon-h-align: calc(((2rem + 2px - 0.9rem) / 2) * -1)`?